src/book/03-guides/03-hardware/01-working-with-robots.mdx

```diff
@@ -37,63 +31,44 @@ To align an ankle correctly perform the following instructions:
+- Remove the arm from the shoulder socket, it will still be attatched by a wire
+- Take off the long part of plastic between the shoudler and neck, it is held by two screws. This piece of plastic can be seen in the following photo:
+  ![shoulder cover](./images/shoulder/shoulder.jpg 'shoulder cover')
+- Now that the piece is off, there is a port for the chord that is still keeping the arm in place, detach this and the arm will be completely detatched
+
+2. Detatching the shoulder
+   - Looking in the shoulder socket there is another circle of screws, remove these and the shoulder will detatch
+     Note: Two plastic spacers should also fall out
+3. Removing the servo
+
+   - Remove and disconnect the boards and fans neccessary to access the shoulder servo from the inside, make sure to note any chords you detatch
+   - On the outside of the robot where the shoulder socket detatched, remove the four screws mounting the servo to the robots body
+   - There is a plastic holder inside the robot holding each shoulder servos in place, slide this towards the robots back to unclip the servos to allow it to fall freely out of its slot
+
+4. Mechanically testing the servo
+   - Get an screw driver that fits the middle screw in the servo horn. Rotate the hornto test if the servo rotates freely without any clunks. If it doesn't rotate or is clunky, it needs replacing
+5. Installing a new servo
+
+   - Get the new servo set with the same serial number on the box (e.g. MX64 or MX106)
+   - Connect the metal circular disk (horn) that came in the box on the new servo, ensuring you line up the indents on the metal disk and servo
+   - Before placing the servo into the robot it needs to be calibrated. Using the dynamixel wizard, confirm the correct baud rate and ID of the servo. You can also check that the servo is properly calibrated by setting the goal position to a couple of different values (0 degress, +90 degrees, -90 degrees) and ensuring the arrow on the horn (the three dots) are pointing in the correct direction. It is also worth testing that the servo behaves correctly around the 180 degree point (wheel mode), if you set the goal position to 170 degrees and then to 190 degrees it should take the long way around to get there.
+   - Place the servo into the body, put back the plastic holder and reconnect all the cords that were disconnected in the 'removing servo' step
+   - Rescrew the four screws that were taken out
+   - The arm of the robot should be staight and perpendicular to the body when the shoulder is set to 0 degrees. This means you need to orientate the server before placing the shoulder back on.
+     On the horn, there are three small indents in the metal, which form an arrow. The point of the arrow needs to be towards the front of the robot, and the two dots towards its back, this ensures the robots shoulder will be correctly aligned.
```

Comment:
I think this part is misleading/contradictory when compared with the next section (and the picture). Best to leave this part to the next section since this actually relates to attaching the shoulder socket

Reply:
I moved it to the next section and also changed the text to match the picture.